omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingSubResource.java

```diff
@@ -238,4 +239,12 @@ public SimpleObject asRef(T delegate) throws ConversionException {
 		return convertDelegateToRepresentation(delegate, description);
 	}
 	
+	/**
+	 * @see SubResource#p[ut(java.lang.String, SimpleObject, RequestContext)
+	 */
+	@Override
+	public void put(String parentUniqueId, SimpleObject post, RequestContext context) throws ResponseException {
+		throw new ResourceDoesNotSupportOperationException();

```

Comment:
the spec asked for put to be used to set no known allergies, without this method we could not implement it